doc/en/developer/source/cite-test-guide/index.rst

```diff
@@ -170,59 +463,41 @@ Run WMS 1.1 tests
       * Ensure ``GML FeatureInfo`` is *checked*
       * Ensure ``Fees and Access Constraints`` is *checked*
       * For ``BoundingBox Constraints`` ensure ``Either`` is selected
-     
+
    #. Click ``OK``
 
-   .. image:: tewms-1_1a.jpg
+   .. image:: ./image/tewms-1_1a.png
 
-   .. image:: tewms-1_1b.jpg
+   .. image:: ./image/tewms-1_1b.png
 
-Run WCS 1.1 tests
+Run WCS 1.0 tests
 -----------------
 
-#. Start GeoServer with the ``citewcs-1.1`` data directory.
-
-#. Change directory back to the cite tools and run the tests::
+#. Prepare the environment:
 
-     ant wcs-1.1
-     
-   With the following parameters:
-   
-   #. ``Capabilities URL``:
-
-         http://localhost:8080/geoserver/wcs?service=wcs&request=getcapabilities&version=1.1.1
-   
-   Click ``Next``
+  - Start GeoServer with the citewcs-1.0 data directory. Example:
 
-   .. image:: tewcs-1_1a.jpg
+   .. code-block:: shell
 
-#. Accept the default values and click ``Submit``
+    cd <root of geoserver install>
+    export GEOSERVER_DATA_DIR=<root of geoserver sources>/data/citewcs-1.0
```

Comment:
Still points to the old, non existing directory.

Reply:
solved in https://github.com/randomorder/geoserver/pull/3/commits/91da4cbdc686ba5397014ab7887381ee240f8a1b